frontend/app/routes/public/renew/$id/ita/confirmation.tsx

```diff
@@ -164,7 +164,7 @@ export default function RenewFlowConfirm() {
             event.preventDefault();
             window.print();
           }}
-          data-gc-analytics-customclick="ESDC-EDSC:CDCP Online Application Form-Adult:Print top - Application successfully submitted click"
+          data-gc-analytics-customclick="ESDC-EDSC:CDCP Online Application Form-ITA:Print top - Confirmation click"
```

Comment:
To match the page title and to be consistent with other flows:
```suggestion
          data-gc-analytics-customclick="ESDC-EDSC:CDCP Online Application Form-ITA:Print top - Your renewal for the Canadian Dental Care Plan is complete click"
```

(Same comment for the rest of the attribute values in this route)